bancor_research/bancor_simulator/v3/spec/actions.py

```diff
@@ -342,9 +342,8 @@ def process_withdrawal(
             bnbnt_renounced = bnbnt_rate * bnt_renounced
 
             state.decrease_pooltoken_balance("bnbnt", bnbnt_renounced)
```

Comment:
this should be changed to `state.decrease_pooltoken_balance("bnt", bnbnt_renounced)`

Reply:
Fixed